Apps/W1/EDocumentConnectors/Tietoevry/app/src/ConnectionSetupCard.Page.al

```diff
@@ -0,0 +1,138 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Tietoevry;
+
+using System.Telemetry;
+
+page 6392 "Connection Setup Card"
+{
+    PageType = Card;
+    SourceTable = "Connection Setup";
+    ApplicationArea = Basic, Suite;
+    UsageCategory = None;
+    Caption = 'Tietoevry Connection Setup';
+    Permissions = tabledata "Connection Setup" = rm;
+    DeleteAllowed = false;
+    InsertAllowed = false;
+
+    layout
+    {
+        area(Content)
+        {
+            group(General)
+            {
+                field(ClientID; this.ClientID)
+                {
+                    Caption = 'Client ID';
+                    ToolTip = 'Specifies the client ID.';
```

Comment:
Move tooltips to table.

Reply:
Since when do we add tooltips on table fields?

Reply:
Moved them to the table, when applicable. Still some tooltips left on the page, since the source is local variables in those cases.